week2.Rmd

```diff
@@ -0,0 +1,130 @@
+$\sigma = Stephan-Boltzmann  \  constant\  (5.67x 10^{-8}\ W/m^2/T^4)$
+$ T= Temperature\ (degrees\  Kelvin^4)$
+
+The Stefan-Boltzmann equation creates a proportional relationship between energy ($\frac{W}{m^2}$) and temperature (kelvins) to the fourth power. compares an energy (power per unit area squared, with units in $\frac{W}{m^2}$) to temperature raised to the fourth (with units in $Kelvin^4$). 
+
+We briefly touched on emissivity in the last problem set, but we'll be delving into that more this week. As a reminder, emissivity refers to the effectiveness of an object to emit an energy. For theoretical black bodies, which absorb all incident heat radiation, emissivity is equal to 1.
+
+**Albedo** is a term that refers to the amount of light that a planet (or any surface) reflects away. This makes albedo kind of like an "opposite" to emissivity. Therefore, a body that perfectly reflects all light (like one covered in snow or ice) would have an albedo of 100% (usually written in decimal form to be 1.00). In contrast, a planet that has a dark surface (or a black body) would have an albedo of 0. And just like emissivity, albedo is unitless. 
+
+We relate albedo to emissivity through the equation: 
+$$ e = 1 - albedo $$
+Let's stop and think about if this equation makes sense -> 
+-We expect a blackbody to have an albedo of zero and an emissivity of 1 since all light is absorbed and none is reflected. e = 1-0, e = 1. Great! 
+-We expect a perfect reflector to have an albedo of one and an emissivity of zero since no light is absorbed and all is reflected. e = 1-1, e = 0. Awesome! 
+
+Now, let's rewrite the Stefan-Boltzmann Law to include albedo: 
+$$ \frac {P}{A} = (1-albedo)\ \sigma T^4 $$
+We'll be using albedo to calculate a more accurate temperature for the earth, plot how different albedos affect Earth's temperature, and think about Earth's climate. 
+
+##Example
+First, we'll start by using albedo to explore the difference in temperature on Mars if it were mostly covered in ice (some scientists think it was once an icy planet!) versus in its current red, sandy state. We'll be using a version of the Stefan-Boltzmann law where we'll solve for Temperature, but this week we'll substitute our $1-albedo$ term in for the emissivity. Here's what the initial equilibrium equation looks like: 
+$$(1-albedo)*So*\pi R^2 = \sigma T^4*4\pi R^2$$
+This equation says that the radiation received by a planet (now affected by the albedo of that planet) is equal to the radiation emitted by that planet. 
+
+Here is the equation above solved for temperature (kelvins):
+$$T = [(1-albedo)*So/4\sigma]^{1/4}$$
+We'll use a solar constant for Mars of 591 $\frac {W}{m^2}$, the Stefan Boltzmann constant of $5.67x 10^{-8}$ and albedos of 0.70 for icy Mars and 0.15 for modern Mars.
+
+This week, you'll be introduced to a new coding tool that makes it easy to define and plot data. We'll first use tibbles to create a data frame with columns and rows. 
```

Comment:
maybe rephrase as "...you'll be introduced to a new coding tool, called a tibble, that makes..."

Reply:
or is the new tool ggplot? Either way, you could mention it here :)